### PR TITLE
Add support for running external pipeline on multiple datasets

### PR DESCRIPTION
**Pull request by cjh1**
Please provide an overview of what this pull request does, and reference any
relevant issues that are addressed. Once submitted you are encouraged to seek
review of the code, and to check the continuous integration results.

By submitting this pull request I agree to the terms of the
[Developer Certificate or Origin][dco].

  [dco]: https://developercertificate.org/

### CODE REVIEW

File: tomviz/python/tomviz/cli/__init__.py [hunk 1]
```diff
@@ -59,23 +65,46 @@ def main(data_file_path, state_file_path, output_file_path, progress_method,
...
-        data_file_path = filenames[0]
+        data_path = filenames[0]
         # fileName is relative to the state file location, so convert to
         # absolute path.
-        data_file_path = os.path.abspath(
-            os.path.join(os.path.dirname(state_file_path), data_file_path))
-        if not data_file_path.lower().endswith('.emd'):
+        data_path = os.path.abspath(
+            os.path.join(os.path.dirname(state_file_path), data_path))
+        if not data_path.lower().endswith('.emd'):
```
Comment (psavery) [inline tomviz/python/tomviz/cli/__init__.py]: So I noticed that when it is reading from the state file, it requires the file to be EMD. But below ([this block](https://github.com/OpenChemistry/tomviz/pull/1928/files#diff-f4a51f1c4f8109ed11a0e42e6aac1c32R96)), there doesn't seem to be an EMD requirement. Is that intentional?

Comment (cjh1) [reply tomviz/python/tomviz/cli/__init__.py]: Well, in the case of a directory being provided we glob for *.emd, in the other case we would try to open the file as an hdf5 file which, will probably raise an exception. i can add in another check for the file ending if you think this is useful.

Comment (psavery) [reply tomviz/python/tomviz/cli/__init__.py]: No, I was just wondering. It's up to you. Looks good otherwise.

Comment (cjh1) [reply tomviz/python/tomviz/cli/__init__.py]: I moved the check further down, so now it should catch both cases.

**Pull request merged**